### DISCUSSION TITLE
Add an option to add custom filters.

### DISCUSSION THREAD

**Discussion by fluentmoheshwar** [ThioJoe/YT-Spammer-Purge#598] [Ideas]
Add an option to add txt files as filter. Contributors can create filters for their language. Users can add them from their repository.
This may fix issue #477, #496 and #529.

**Comment by ThioJoe** [comment]
You can pretty much already do this using the filter modes "scan usernames / comment text / usernames and comment text for criteria you choose", unless you mean something else

**Reply by fluentmoheshwar** [reply]
Yes, but that isn't enough for a whole language. I have to enter them manually every time.

**Reply by ThioJoe** [reply]
> Yes, but that isn't enough for a whole language. I have to enter them manually every time.

That's what the config file is for, so you don't have to enter it every time

**Reply by fluentmoheshwar** [reply]
I use that config. But i don't know how to do that. Teach me.

**Reply by TechStudent10** [reply]
@biswasmoheshwar Found this in the wiki https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Config-File-Settings

**Comment by fluentmoheshwar** [comment]
Found! Thanks.
